### PR TITLE
Update community-css-themes.json

### PR DESCRIPTION
**Pull request by chad-bennett**
add Al Dente theme

### CODE REVIEW

**Comment by ericaxu** [comment]
The repo should be your {github_name/repo_name}, please see the theme above yours. It should be `chad-bennett/al-dente-obsidian-theme`.

Other than that, I would suggest fixing the indentation on line 331. Thanks!

**Comment by chad-bennett** [comment]
updated. thanks.

File: community-css-themes.json [hunk 1]
```diff
@@ -327,5 +327,12 @@
         "repo": "radekkozak/discordian",
         "screenshot": "media/screenshots/discordian-full-mode.png",
         "compatibleModes": ["dark"]
+    },
+    {
+        "name": "Al Dente",
+        "author": "chad-bennett",
+        "repo": "chad-bennett/al-dente-obsidian-them",
```
Comment (heliostatic) [inline community-css-themes.json:334]:
```suggestion
        "repo": "chad-bennett/al-dente-obsidian-theme",

```

Comment (chad-bennett) [reply community-css-themes.json:334]: good catch. fixing it.

**Comment by chad-bennett** [comment]
fix typo.

**Comment by ericaxu** [comment]
Looking great, thanks!

**Pull request merged**